src/components/UI/Form/EmojiInput/EmojiInput.tsx

```diff
@@ -48,7 +48,7 @@ const customSuggestionsFilter = (searchValue: string, suggestions: Array<any>) =
   return filteredSuggestions.slice(0, length);
 };
 
-const DraftField = React.forwardRef((inputProps: any, ref: any) => {
+const DraftField = (inputProps: any) => {
```

Comment:
I am not sure about these changes? Is this related to the import functionality?

Reply:
The focus was somehow **shifting** on  **Emoji** Input when the user is typing elsewhere in the form. And we mostly using **ref** to focus emoji input. And I did not find a use case wherein we could potentially want to focus Emoji input.

Reply:
Ok